### PR TITLE
Show org config fields to admins as read only fields

### PR DESCRIPTION
**Pull request by norkans7**
This will allow Ureport to display org config that the admin can not change but let them know in case  the config mismatch the fields on RapidPro

### CODE REVIEW

File: dash/orgs/views.py [hunk 1]
```diff
@@ -248,7 +248,8 @@ def get_form(self):
             # add all our configured org fields as well
             config_fields = getattr(settings, 'ORG_CONFIG_FIELDS', [])
             for config_field in config_fields:
-                if is_super or not config_field.get('superuser_only', False):
+                read_only = config_field.get('read_only', False)
+                if is_super or read_only or not config_field.get('superuser_only', False):
```
Comment (rowanseymour) [inline dash/orgs/views.py]: not related to this change - but how useful is `superuser_only` - shouldn't we just let superusers see everything?

Comment (norkans7) [reply dash/orgs/views.py]: That is the case `superuser_only` allows only the superuser to edit

File: dash/orgs/views.py [hunk 2]
```diff
@@ -260,6 +261,10 @@ def get_form(self):
                     else:
                         form.fields[field_name] = forms.CharField(**config_field['field'])
 
+                    if not is_super and read_only:
+                        form.fields[field_name].widget.attrs['readonly'] = 'readonly'
```
Comment (rowanseymour) [inline dash/orgs/views.py]: does this do anything to prevent a user posting data for a field manually?

Comment (norkans7) [reply dash/orgs/views.py]: Ohh right, I'll add that

**Pull request merged**